packages/fxa-auth-server/config/index.js

```diff
@@ -702,6 +702,16 @@ const conf = convict({
       default: 3 * 31 * 24 * 60 * 60, // ~3 months
       env: 'STRIPE_CUSTOMER_CACHE_TTL_SECONDS',
     },
+    transactionalEmails: {
```

Comment:
This isn't used yet, but is the groundwork for FXA-1148. It will be used to enable / disable the rest of the email logic added for the epic.